apps-rendering/src/components/liveblogHeader.tsx

```diff
@@ -34,6 +37,29 @@ const headerBackgroundStyles = ({
 	}
 `;
 
+const isBlog = (format: Format): boolean =>
+	format.design === Design.LiveBlog || format.design === Design.DeadBlog;
+
+const lineStyles = (format: Format): SerializedStyles =>
+	css`
+		${from.wide} {
+			width: ${wideContentWidth}px;
+			margin-left: auto;
+			margin-right: auto;
+		}
+		div {
+			${darkModeCss`
+			background-image: repeating-linear-gradient(
+				to bottom,
+				${isBlog(format) ? neutral[93] : neutral[20]},
+				${isBlog(format) ? neutral[93] : neutral[20]} 1px,
```

Comment:
Just as a suggestion, it may be better if we don't repeat this bit `${isBlog(format) ? neutral[93] : neutral[20]}` and have it as a variable within the lineStyles function

Reply:
Good shout, updated.